File: FlexServ Backend/SpringBoot/BusinessService/src/main/java/com/flexserv/dto/request/UpdateProviderRequest.java
package com.flexserv.dto.request;

import java.util.Set;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class UpdateProviderRequest {

    private String userName;

    private String userPhone;

    private String companyName;

    private Integer experienceYears;

    private String bio;

    private Set<Long> serviceIds;
}

File: FlexServ Backend/DotNet/AdminService/Dtos/UpdateProviderRequest.cs
using System.Collections.Generic;

namespace AdminService.Dtos
{
    public class UpdateProviderRequest
    {
        public string? UserName { get; set; }
        public string? UserPhone { get; set; }
        public string? CompanyName { get; set; }
        public int? ExperienceYears { get; set; }
        public string? Bio { get; set; }
        public HashSet<long>? ServiceIds { get; set; }
    }
}
